README.md

```
@@ -231,20 +231,32 @@ ftwd create-chips fields.parquet --reproject
 
 Create raster masks from vector boundaries for each grid cell. Outputs Cloud Optimized GeoTIFFs (COGs).
 
+Masks are written into per-chip directories, matching the catalog structure `create-dataset` produces:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)